graphene_django/relay/connection.py

```diff
@@ -76,46 +74,76 @@ def connection_from_sized_sliceable(
     else:
         raise ValueError(f"Unreachable: {args}")
 
-    first_edge_cursor: Optional[str] = edges[0].cursor if edges else None
-    last_edge_cursor: Optional[str] = edges[-1].cursor if edges else None
-
-    return connection_type(
-        edges=edges,
-        pageInfo=page_info_type(
-            startCursor=first_edge_cursor,
-            endCursor=last_edge_cursor,
-            hasPreviousPage=has_previous_page,
-            hasNextPage=has_next_page,
-        ),
-    )
+    def create_connection(handle_result: Tuple[List[Edge], bool, bool]):
+        (
+            edges,
+            has_previous_page,
+            has_next_page,
+        ) = handle_result
+
+        first_edge_cursor: Optional[str] = edges[0].cursor if edges else None
+        last_edge_cursor: Optional[str] = edges[-1].cursor if edges else None
```

Comment:
Just realized: does this repo use typings?

Reply:
No it does not, unfortunately. I'm trying to add some little by little